### ISSUE TITLE
Builtin object methods have wrong attributes

### ISSUE THREAD

**Issue by HalidOdat** [boa-dev/boa#761]
**Describe the bug**
Builtin object methods like `Object.is()` `Array.prototype.push`, etc, have `WRITABLE`, `ENUMERABLE` and `CONFIGURABLE`, but they should have `WRITABLE`, `NON_ENUMERABLE` and `CONFIGURABLE`

**To Reproduce**
Steps to reproduce the issue, or JavaScript code that causes this failure.

This JavaScript code reproduces the issue:
```javascript
Array.prototype.propertyIsEnumerable("push") // this returns `true`, but it should return `false`
```

**Comment by Razican** [comment]
Hey, any mention on when was this solved / why was it closed?

**Comment by HalidOdat** [comment]
> Hey, any mention on when was this solved / why was it closed?

This was closed when #722 got merged, because they have the correct attribute now https://github.com/boa-dev/boa/blob/master/boa/src/object/mod.rs#L829-L833 and https://github.com/boa-dev/boa/blob/master/boa/src/object/mod.rs#L856-L860 in `ConstructorBuilder` `.method` and `.static_method` which removed the need for make_builtin_fn and `make_constructor_fn` and manual object/prototype creation.
